Differential Equations.tex

```diff
@@ -284,10 +284,13 @@ \subsubsection*{Solution}
 \item Plug in the ICs to get $a_n, b_n$.
 
 \begin{gather*}
-a_n = \frac{1}{l} \int_{-l}^{l} f(x) \sin(\frac{n\pi}{l}x) dx \\
+a_n = \frac{1}{l} \int_{-l}^{l} f(x) \cos(\frac{n\pi}{l}x) dx \\
```

Comment:
Interesting... I was confused by this vs. the definition of `T(t)`. Why is it `sine`?